aws/cloudformation/metaflow-cfn-template.yml

```diff
@@ -1195,14 +1211,14 @@ Resources:
         MaxvCpus: !Ref MaxVCPUBatch
         SecurityGroupIds:
           - !GetAtt VPC.DefaultSecurityGroup
-        Type: EC2
+        Type: !If [BatchFargate, 'FARGATE', 'EC2']
         Subnets:
           - !Ref Subnet1
           - !Ref Subnet2
-        MinvCpus: !Ref MinVCPUBatch
-        InstanceRole: !GetAtt 'ECSInstanceProfile.Arn'
-        InstanceTypes: !Ref ComputeEnvInstanceTypes
-        DesiredvCpus: !Ref DesiredVCPUBatch
+        MinvCpus: !If [BatchFargate, !Ref AWS::NoValue, !Ref MinVCPUBatch]
```

Comment:
We would need the same conditional for `MaxvCpus`?

Reply:
Actually, oddly enough, no.  MaxVCPUs is still a required parameter, as I think the intention was to limit clusters from overscaling.  Minimum and desired aren't applicable since we don't have resource on standby.

Reply:
Makes sense